### DISCUSSION TITLE
Are web.archive.org links allowed?

### DISCUSSION THREAD

**Discussion by upintheairsheep** [microsoft/winget-pkgs#102943] [Q&A] [answered]
Are links to the wayback machine allowed on WinGet-PKGs, as I can both get the installers of discontinued but still functional programs, and get stuff for programs that have session-based or temporary links.

**Comment by Trenly** [comment, marked as answer]
> Are links to the wayback machine allowed on WinGet-PKGs, as I can both get the installers of discontinued but still functional programs, and get stuff for programs that have session-based or temporary links.

Unfortunately they are not allowed since packages must come from the ISV or an official / authorized mirror

**Reply by upintheairsheep** [reply]
Thank you, however I believe it should be allowed due to its permanence.

**Reply by Trenly** [reply]
Permanence is not the only consideration that needs to be made. Security of the package, publisher wishes, terms and conditions of the sources, terms and conditions of the package, etc. all come into play

**Comment by SpecterShell** [comment]
According to their [Terms of Use](https://archive.org/about/terms.php):

> Access to the Archive’s Collections is provided at no cost to you and is granted for scholarship and research purposes only.
